glean-core/android/src/main/java/mozilla/telemetry/glean/scheduler/PingUploadWorker.kt

```diff
@@ -98,14 +95,9 @@ class PingUploadWorker(context: Context, params: WorkerParameters) : Worker(cont
      */
     @Suppress("ReturnCount")
     override fun doWork(): Result {
-        // We counte the number of failures,
-        // if we get more than three failures in upload we will stop this worker.
-        //
-        // This is a hack before a more robust mechanism is implemented in Rust.
-        var uploadFailures = 0
+        // Create a slot of memory for the task: glean-core will write data into
```

Comment:
Where does the creation happen? Are you referring to the `val incomingTask = ...` below? If so, please move the comment closer.

Reply:
Hm, this only shows as an added line bcause I moved the lines around it.

I see I did move the lines around it incorrectly and it should be right before `val incomingTask = ...`. Good catch!